io.catenax.id_based_comment/1.0.0/IdBasedComment.ttl

```diff
@@ -0,0 +1,184 @@
+   samm:description "The Comment ID uniquely identifies the entity within the business relationship between a customer and its supplier."@en ;
+   samm:characteristic :UUIDv4IdTrait ;
+   samm:exampleValue "f5c151e4-30b5-4456-94fd-2a7b559b6121" .
+
+:objectId a samm:Property ;
+   samm:preferredName "Object ID"@en ;
+   samm:description "The ID of the object to which the comment belongs."@en ;
+   samm:characteristic :UUIDv4IdTrait ;
+   samm:exampleValue "dfeb1334-497e-4dab-97c1-4e6f4e1c0320" .
+
+:author a samm:Property ;
+   samm:preferredName "Author"@en ;
+   samm:description "The E-Mail address of the creator who wrote the comment. In case comment creator wants to stay anonymous, show BPNL in app which is sent the comment along."@en ;
+   samm:characteristic :AuthorCharacteristic ;
+   samm:exampleValue "someone@company.com" .
+
+:postedAt a samm:Property ;
+   samm:preferredName "Posted at"@en ;
+   samm:description "Point in time when the comment was created."@en ;
+   samm:characteristic :Timestamp ;
+   samm:exampleValue "2023-03-10T12:27:11.320Z"^^xsd:dateTimeStamp .
+
+:changedAt a samm:Property ;
+   samm:preferredName "Changed at"@en ;
+   samm:description "Point in time when the comment was last changed."@en ;
+   samm:characteristic :Timestamp ;
+   samm:exampleValue "2023-03-10T12:27:11.320Z"^^xsd:dateTimeStamp .
+
+:commentText a samm:Property ;
+   samm:preferredName "Comment Text"@en ;
+   samm:description "Text up to 5.000 characters in format \"UTF-8\" will be shared."@en ;
+   samm:characteristic :CommentTrait ;
+   samm:exampleValue "Hello, this is a comment!" .
+
+:commentType a samm:Property ;
+   samm:preferredName "Comment Type"@en ;
+   samm:description "Type of a comment to indicate a priority."@en ;
+   samm:characteristic :CommentTypeCharacteristic .
+
+:requestDelete a samm:Property ;
+   samm:preferredName "Request Delete"@en ;
+   samm:description "Indicates that the deletion of the comment is requested by sender incl. all of it's history."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:listOfReferenceDates a samm:Property ;
+   samm:preferredName "List Of Reference Dates"@en ;
+   samm:description "Point in time the comment belongs to."@en ;
+   samm:characteristic :ListOfReferenceDatesCharacteristic ;
+   samm:exampleValue "2023-11-05"^^xsd:date .
+
+:objectType a samm:Property ;
+   samm:preferredName "Object Type"@en ;
+   samm:description "Aspect Model which the comment refers to. The value MUST consist of the Catena-X aspect model unique identifier of the referenced data model without a version."@en ;
+   samm:characteristic :ObjectTypeCharacteristic ;
+   samm:exampleValue "urn:samm:io.catenax.week_based_capacity_group" .
+
+:customer a samm:Property ;
+   samm:preferredName "Customer"@en ;
+   samm:description "The Business Partner Number Legal Entity (BPNL) of the party requesting materials from a supplier."@en ;
+   samm:characteristic :BPNLTrait ;
```

Comment:
Here also please check if you can make use of shared aspect model https://github.com/eclipse-tractusx/sldt-semantic-models/blob/a07bdd5b1515767d4f8d07deb47cf2a4aa728bef/io.catenax.shared.business_partner_number/1.0.0/BusinessPartnerNumber.ttl

Reply:
I love it, this would make sense for the other models of DCM as well!